src/AuctionHouseBotScript.cpp

```diff
@@ -24,14 +24,18 @@ class AHBot_WorldScript : public WorldScript
         if (HasPerformedStartup == true)
         {
             LOG_INFO("server.loading", "AuctionHouseBot: (Re)populating item candidate lists ...");
-            auctionbot->PopulateItemCandidatesAndProportions();
         }
     }
 
     void OnStartup() override
     {
         LOG_INFO("server.loading", "AuctionHouseBot: (Re)populating item candidate lists ...");
         auctionbot->PopulateItemCandidatesAndProportions();
+        if (sConfigMgr->GetOption<bool>("AuctionHouseBot.Seller.UseDBDropRates.Enabled", true))
+        {
+            auctionbot->PopulateQuestRewardItemIDs();
+            auctionbot->PopulateItemDropChances();
+        }
```

Comment:
Similar to the other comment, you'll need this in OnAfterConfigLoad due to the ".reload config" command.  That's the typical way server administrators reload their configs while the server is running.